db/mongodb.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose');
+
+mongoose.set('useCreateIndex', true);
+
+mongoose
+	.connect(
+		`mongodb+srv://HYF-Ammar:${process.env.MongoDB_password}@cluster0-ptw2q.azure.mongodb.net/${process.env
```

Comment:
if we have the environment variable called, for example, DATABASE_URL with all the information about the data, we could use it here so we would have:

.connect(`${DATABASE_URL}`)

If you think about it, usually you are not going to use the password and database name in different places so that's another reason why you can keep it only in one enviornment variable.